Content.Shared/SS220/ChameleonStructure/ChameleonStructureComponent.cs

```diff
@@ -0,0 +1,51 @@
+// © SS220, An EULA/CLA with a hosting restriction, full text: https://raw.githubusercontent.com/SerbiaStrong-220/space-station-14/master/CLA.txt
+
+using Content.Shared.Whitelist;
+using Robust.Shared.GameStates;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization;
+
+namespace Content.Shared.SS220.ChameleonStructure;
+
+/// <summary>
+///     Allow players to change sctructure sprite to any other structure prototype.
+/// </summary>
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentState(true)]
+[Access(typeof(SharedChameleonStructureSystem))]
+public sealed partial class ChameleonStructureComponent : Component
+{
+    /// <summary>
+    ///     EntityPrototype id that chameleon item is trying to mimic.
+    /// </summary>
+    [ViewVariables(VVAccess.ReadOnly)]
+    [DataField, AutoNetworkedField]
+    public EntProtoId? Default;
```

Comment:
`Default` подошло бы в случае, если бы ты не менял его в системе (условно константа)
```suggestion
    public EntProtoId? Prototype;
```